.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+// https://prettier.io/, for embedded configuration files
+module.exports = {
+    useTabs: true
```

Comment:
tabs vs spaces! 😂
https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/

Reply:
😂  not to start a fight here -- is this a convention (at shopify)?

Reply:
Tabs are enforced by SOPS when writing JSON: https://github.com/mozilla/sops/blob/2395f07610e45d507ec0d4b3ad48dbf502ed5bed/stores/json/store.go#L220 .  The other files don't matter.

I'm impartial to the style, I'm just after consistency and a stable autoformat.